docs/changelog/96186.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 96186
```

Comment:
No need to introduce a changelog for test changes.
I also changed the label from bug to test.

Reply:
Gotcha